newrelic-fluent-bit-output.yaml

```diff
@@ -0,0 +1,44 @@
+package:
+  name: newrelic-fluent-bit-output
+  version: 1.16.0
+  epoch: 0
+  description: A Fluent Bit output plugin that sends logs to New Relic
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - busybox
+      - build-base
+      - go
+      - ca-certificates-bundle
+
+pipeline:
+  # We can't use go/install because this requires specific ldflags to set the version
+  - uses: git-checkout
+    with:
+      repository: https://github.com/newrelic/newrelic-fluent-bit-output
+      tag: v${{package.version}}
+      expected-commit: 8e3aa7c9d829d2c36d8007c2d44295cefd5ee284
+
+  - if: ${{build.arch}} == 'x86_64'
+    runs: |
+      make VERSION=v${{ package.version }} linux/amd64
+      mkdir -p ${{targets.destdir}}/newrelic/newrelic-fluent-bit-output
+      cp ./out_newrelic-linux-amd64-v${{ package.version }}.so ${{targets.destdir}}/newrelic/newrelic-fluent-bit-output
+
+  - if: ${{build.arch}} == 'aarch64'
+    runs: |
+      make VERSION=v${{ package.version }} linux/arm64
+      mkdir -p ${{targets.destdir}}/newrelic/newrelic-fluent-bit-output
+      cp ./out_newrelic-linux-arm64-v${{ package.version }}.so ${{targets.destdir}}/newrelic/newrelic-fluent-bit-output
```

Comment:
I think we should make this one step, that sets `ARCH=$(go env GOARCH)` and passes that around instead of `amd64` and `arm64`.

The reason is, when we add a third arch, we'll want to make sure we actually produce a useful apk for it, without having to check every package for `if`s like this.

It looks like it should be possible to do, let me know if you hit a snag.

Reply:
brilliant idea, thanks!